crates/aranya-client/src/client.rs

```diff
@@ -262,4 +292,67 @@ impl Team<'_> {
             .revoke_label(context::current(), self.id, device, label)
             .await??)
     }
+
+    /// Query devices on team.
+    pub async fn query_devices_on_team(&mut self) -> Result<Vec<DeviceId>> {
+        Ok(self
+            .client
+            .daemon
+            .query_devices_on_team(context::current(), self.id)
+            .await??)
+    }
+
+    /// Query device role.
+    pub async fn query_device_role(&mut self, device: DeviceId) -> Result<Role> {
+        Ok(self
+            .client
+            .daemon
+            .query_device_role(context::current(), self.id, device)
+            .await??)
+    }
+
+    /// Query device keybundle.
+    pub async fn query_device_keybundle(&mut self, device: DeviceId) -> Result<KeyBundle> {
+        Ok(self
+            .client
+            .daemon
+            .query_device_keybundle(context::current(), self.id, device)
+            .await??)
+    }
+
+    /// Query device label assignments.
+    pub async fn query_device_label_assignments(&mut self, device: DeviceId) -> Result<Vec<Label>> {
+        Ok(self
+            .client
+            .daemon
+            .query_device_label_assignments(context::current(), self.id, device)
+            .await??)
+    }
+
+    /// Query AFC network ID.
+    pub async fn query_afc_net_identifier(&mut self, device: DeviceId) -> Result<NetIdentifier> {
+        Ok(self
+            .client
+            .daemon
+            .query_afc_net_identifier(context::current(), self.id, device)
+            .await??)
+    }
+
+    /// Query AQC network ID.
+    pub async fn query_aqc_net_identifier(&mut self, device: DeviceId) -> Result<NetIdentifier> {
+        Ok(self
+            .client
+            .daemon
+            .query_aqc_net_identifier(context::current(), self.id, device)
+            .await??)
+    }
+
+    /// Query label exists.
+    pub async fn query_label_exists(&mut self, label: Label) -> Result<bool> {
```

Comment:
All of the queries are named `query_foo` or `query_bar`. They're a little unnatural. E.g.:

```rust
if client.query_label_exists(FOO).await? { ... }
```

We could do what we've done elsewhere and add

```rust
pub struct Queries<'a> {
    client: &'a mut Client,
}

impl Queries {
    pub async fn aqc_net_identifier(&mut self, device: DeviceId) -> Result<NetIdentifier> { ... }
    pub async fn label_exists(&mut self, label: Label) -> Result<bool> { ... }
```